i6_models/parts/lstm.py

```diff
@@ -0,0 +1,70 @@
+__all__ = ["LstmBlockV1Config", "LstmBlockV1"]
+
+from dataclasses import dataclass
+import torch
+from torch import nn
+from typing import Dict, Tuple, Union
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class LstmBlockV1Config(ModelConfiguration):
+    input_dim: int
+    hidden_dim: int
+    num_layers: int
+    bias: bool
+    dropout: float
+    bidirectional: bool
+    enforce_sorted: bool
+
+    @classmethod
+    def from_dict(cls, model_cfg_dict: Dict[str, Any]):
```

Comment:
I don't see this for other part configs, why do we need this here?

Reply:
I use it in the model definition to conert a dict to the config class. Need might be a bit strong but I like it :D